recipes-kernel/intelsgx/intelsgx.bb

```diff
@@ -0,0 +1,29 @@
+SUMMARY = "Intel(R) SGX Data Center Attestation Primitives driver"
+LICENSE = "BSD-3-Clause | GPL-2.0"
+LIC_FILES_CHKSUM = "file://License.txt;md5=633d1f5182ada7cd064194532a4a79d4"
+
+DEPENDS += " virtual/kernel"
+
+inherit module
+
+SRC_URI  = "git://git@github.com/intel/SGXDataCenterAttestationPrimitives.git"
```

Comment:
Do not leave git@ username in here, add ;protocol=https.

Reply:
done